### ISSUE TITLE
No floor plan

### ISSUE THREAD

**Issue by motahare96** [aqeelanwar/PEDRA#64]
Hi
Thanks for sharing your code. Unfortunately there's a problem with indoor long floor plan as I can't download it. It's necessary for infer mode.
with regards

**Comment by aqeelanwar** [comment]
It should be in the zip file of the environment you downloaded by the name of long_floor.PNG.
Here is the link in case it was misplaced.
https://gtvault-my.sharepoint.com/:u:/g/personal/manwar8_gatech_edu/EYZQqeOIqsFEtaZOGzhzu6oBndlCiAAvDjTwbuh9n3R7jw?e=5VvVRs

**Comment by siddy2000** [comment]
HI Mr. Aqeelan, your completed file of all of the environments seems to be expired.

**Comment by aqeelanwar** [comment]
Hello,

I have updated the link. Let me know if you can access it now.

**Comment by siddy2000** [comment]
Yes that works great. Thanks for everything! Your work is amazing.

**Comment by motahare96** [comment]
Thanks for your response but .png file can't be downloaded.

**Comment by aqeelanwar** [comment]
@motahare96 
Were you able to solve the issue?

**Comment by siddy2000** [comment]
Yes I was. Thank you.
